src/components/interaction-designer/block-editors/choices/ChoiceMappingModal.vue

```diff
@@ -1,14 +1,9 @@
 <template>
    <div class="choice-mapping-modal">
      <div class="form-group">
-       <validation-message
```

Comment:
moved this outside so we will use once single <validation-message for both programmatic & ajv validation